### ISSUE TITLE
What should happen regarding 1.2 Tiny features that are not mentioned in SVG2 spec?

### ISSUE THREAD

**Issue by BigBadaboom** [w3c/svgwg#199]
There are a few references to 1.2 Tiny in the SVG2 spec.  However I have noticed that there are a number of 1.2 Tiny features that AFAICS are not mentioned at all in this spec.  Since 1.2 Tiny reached recommendation status, shouldn't these be mentioned somewhere?  Even if it is only in a deprecated features table?

For example (not exhaustive - just a quick search made)

- `viewport-fill` and `viewport-fill-opacity` properties
- `buffered-rendering` property
- `transform="ref()"`
- `<textArea>`, `<tbreak>`, and related properties
- `editable` on text elements

**Comment by AmeliaBR** [comment]
Tiny 1.2 is mostly being treated as a dead-end fork of SVG.  So consider all of those as deprecated or obsolete.

From what I can tell, most of the references to Tiny in SVG 2 are in the form of annotations about specific features from that spec that we decided to keep.

However, there are a couple back-references to Tiny in the `vector-effects` section and the "Painting vector effects" sections. Probably both need a re-write.  (There's also links to Tiny in the definitions-filters.xml file that we need to update to point to equivalent definitions in SVG 2.  )

Beyond that: If there are specific features from Tiny you want to preserve in SVG 2, that could be considered as a feature request.  From your list, I can say that some are now redundant:

- `viewport-fill` is replaced with CSS `background` properties 
- `buffered-rendering` was dropped in favor of `will-change`
- the multi-line text model has been completely over-hauled so that `<textArea>` is not required

However, a `<tbreak>` element may still prove useful in the future, as could a `editable` option.  

A `ref` type transform, that could reset some or all of the transformation sequence to another SVG, is definitely something I'd be interested in, but it would have to be discussed in the context of the CSS Transforms Level 2 module.

**Comment by BigBadaboom** [comment]
Even if none of those features make it to SVG2. I feel like the spec should at least acknowledge 1.2 Tiny and explain that it is considered a dead fork.  And that SVG2 is a development of 1.1.

At the moment 1.2 is only present as a sort of unacknowledged haunting spirit.  Poor old 1.2 :)

**Comment by AmeliaBR** [comment]
Any suggestions of where to integrate this context? Somewhere in the overview page?

**Comment by BigBadaboom** [comment]
In the Introduction maybe?  Something like:

> 1.4 Relationship to previous versions of this standard
>
> This version 2 edition of the SVG standard has been developed based on, and built upon, the 1.1 edition released in 2003.  An intermediate version of SVG - named 1.2 Tiny - was released in 2008. However it did not receive wide acceptance, and as a consequence, there have been very few implementations of its updated feature set.  Those of its features that have been widely implemented are included as part of this specification.  Otherwise the SVG Working Group consider version 1.2 Tiny to be a deprecated branch of the SVG standard.

**Comment by AmeliaBR** [comment]
@BigBadaboom I like that.

@nikosandronikos @Tavmjong, what do you think?

**Comment by nikosandronikos** [comment]
Good call. I think we definitely need something like this to clarify the relationship.
